File: C/Chapter-07/Question/Question2.c
#include <stdio.h>

int main(){
  int num;
  int start = 1;
  printf("Enter the number: ");
  scanf("%d", &num);

  while(start<=num){
    printf("3 x %d = %d\n", start, start*3);
    start++;
  }

  return 0;
}

File: Java/Chapter-05/Question/Question2.java
public class Question2 {
  public static void main(String[] args) {
    int start = 1;
    while(start <= 100){
      System.out.print(start + " ");
      start++;
    }
    System.out.print("\n");
    start--;
    do{
      System.out.print(start + " ");
      start--;
    }while(start > 0);
  }
}
